docs/install/kubernetes.md

```diff
@@ -214,11 +215,26 @@ In order to opt out of some of the components, you can use the following helm va
 ...
 ```
 
+## Storage resources
+
+When installing the MLRun Community Edition, several storage resources are created:
+
+- **PVs via default configured storage class**: Used to hold the file system of the stacks pods, including the MySQL database of MLRun, Minio for artifacts and Pipelines Storage and more. These are deleted when the stack is uninstalled.
+- **Container Images in the configured docker-registry**: When building and deploying MLRun and Nuclio functions via the MLRun Community Edition, the function images are stored in the given configured docker registry. These images persist in the docker registry and are not deleted.
+
 ## Uninstalling the Chart
+
+The following command will delete the pods, deployments, config maps, services, roles+role bindings and PVCs associated with the chart and release.
```

Comment:
The following command deletes the pods, deployments, config maps, services, roles+role bindings and PVCs associated with the chart and release.